src/Orm/Zed/Test/Persistence/SpyTestQuery.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Orm\Zed\Test\Persistence;
+
+use Orm\Zed\Test\Persistence\Base\SpyTestQuery as BaseSpyTestQuery;
+
+/**
+ * Skeleton subclass for performing query and update operations on the 'spy_test' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements. This class will only be generated as
+ * long as it does not already exist in the output directory.
+ */
+class SpyTestQuery extends BaseSpyTestQuery
```

Comment:
Is guess test is for making some examples for user. But I think better to chose different name, because `test` is usually belongs to testing purposes and it could cause misunderstandings. I'd prefer to use `Example` or `Dummy`.

Reply:
this test will be removed in context of following tickets